portfolio/src/main/java/com/google/sps/servlets/AnalyzeTweet.java

```diff
@@ -43,12 +48,30 @@ public void init() throws ServletException {
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-      //
+      float worstScore = Float.MAX_VALUE;
+      String worstScoreLink = null;
+
       try{
         List<Status> tl = twitter.getUserTimeline(request.getParameter("handle"), new Paging(1, 200));
+        LanguageServiceClient languageService = LanguageServiceClient.create();
+        
         tl.removeIf(status -> status.isRetweet());
-        System.out.println(tl.size());
+        for(Status status:tl){
+            Document doc = Document.newBuilder().setContent(status.getText()).setType(Document.Type.PLAIN_TEXT).build();
+            Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+            float score = sentiment.getScore();
+            worstScoreLink = score < worstScore ? "https://twitter.com/" + status.getUser().getScreenName() + "/status/" + status.getId() : worstScoreLink;
```

Comment:
It might be good to write out an actual if statement here because both this line and the line below are based off of the same conditional.